backend/packages/worker/src/worker/parsing/models.py

```
@@ -0,0 +1,14 @@
+from pydantic import BaseModel, Field
+
+class ParsedPage(BaseModel):
+    page_number: int = Field(..., description="The original page number")
+    text: str = Field(..., description="Extracted text from the page")
+    is_blank: bool = Field(False, description="Whether the page was intentionally blank")
+
+class ParsedArtifact(BaseModel):
+    doc_id: str = Field(..., description="Identifier matching the Tier 1 Document doc_id")
+    pages: List[ParsedPage] = Field(default_factory=list, description="Text broken down by page")
+    total_pages: int = Field(0, description="Total number of parsed pages")
+    is_low_quality: bool = Field(False, description="Flag indicating poor OCR or extraction quality")
```

Ruff (0.15.11)

[warning] Boolean positional value in function call

(FBT003)